include/swift/Runtime/Concurrency.h

```diff
@@ -900,6 +918,20 @@ SerialExecutorRef swift_task_getCurrentExecutor(void);
 SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
 SerialExecutorRef swift_task_getMainExecutor(void);
 
+/// Return the generic (default global concurrent) executor reference.
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+SerialExecutorRef swift_task_getGenericExecutor(void);
```

Comment:
If you're adding these new functions to the runtime ABI, please note which Swift release added them.

But this function in particular I think we don't need — there's an ABI-guaranteed way of constructing this already (IIRC it's the all-zeroes value).

Reply:
Yes, it's `0, 0` so I think I could bitcast cast... I'll give that a go so we can avoid runtime functions here.
I wasn't sure if we want to bake in this assumption but I guess it already is baked in 🤔

Reply:
I did the same for `_getUndefinedTaskExecutor` as well, to avoid runtime calls: https://github.com/apple/swift/pull/68793/commits/1318d3ead1526e596dd302e434d43622d8aa45c0